teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1155 @@
+{
```

Comment:
<p>Just wondering, what happens if the pdb structure is a dimer and contains to equal chains that both contain the ligand with the same name?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>I tried with 6jkh in complex with NAD and it is doing it for both ligands. Curiously it seems to prepare one of them correctly and the other looks wrong .</p><p> </p><p>I do not know if it is able to use the smiles template correctly for two molecules presented as one and if we can make it do so.</p><p>Maybe we could separate the ligand and do the process for each individually.</p>

Reply:
<p>We were able to resolve this. The SMILES string of the ligand needs to be duplicated, then the ligands can be prepared correctly. the rest is not affected</p>